sambaplayersdk/src/main/java/com/sambatech/player/cast/CastObject.java

```diff
@@ -119,13 +121,20 @@ public CastDRM getDrm() {
         return drm;
     }
 
+    public String getLive() {
+        return live;
+    }
 
+    public void setLive(String live) {
+        this.live = live;
+    }
 
     @Override
     public String toString() {
         return "{" +
                 "\"title\":\"" + title + '\"' +
                 ", \"m\":\"" + m + '\"' +
+                (live != null && !live.isEmpty() ? ", \"live\":\"" + live + '\"' : "") +
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS864)